src/SmartComponents/Actions/ActionsOverview.js

```diff
@@ -79,15 +79,15 @@ class ActionsOverview extends Component {
                 <Main>
                     <Grid gutter='md' xl={ 4 } sm={ 6 }>
                         <GridItem>
-                            <Card>
+                            <Card className={ 'pf-t-light  pf-m-opaque-100' }>
```

Comment:
You don't need curly braces here. simple `className='pf-t-light pf-m-opaque-100` will work

Reply:
good to know, thanks!